updates/pgmonitor--1.0.1--2.0.0.sql

```diff
@@ -0,0 +1,156 @@
+
+DROP MATERIALIZED VIEW @extschema@.ccp_stat_bgwriter;
+DELETE FROM @extschema@.metric_views WHERE view_name = 'ccp_stat_bgwriter');
+
+CREATE FUNCTION @extschema@.ccp_stat_bgwriter() RETURNS TABLE
+(
+    buffers_clean bigint
+    , maxwritten_clean bigint
+    , buffers_alloc bigint
+)
+    LANGUAGE plpgsql
+AS $function$
+DECLARE
+BEGIN
+
+IF current_setting('server_version_num')::int >= 170000 THEN
+
+    RETURN QUERY
+    SELECT
+        b.buffers_clean
+        , b.maxwritten_clean
+        , b.buffers_alloc
+    FROM pg_catalog.pg_stat_bgwriter b;
+
+ELSE
+
+    RETURN QUERY
+    SELECT
+        b.buffers_clean
+        , b.maxwritten_clean
+        , b.buffers_alloc
+    FROM pg_catalog.pg_stat_bgwriter b;
+
+END IF;
```

Comment:
Are these queries supposed to be different, or are we supposed to not have the version flag here?

Reply:
Same as other catch